documentation/Using-Azure-TestFramework.md

```diff
@@ -105,17 +106,21 @@ Changing the above properties at run-time has the potential to hard code few thi
 
 Ensure that the `HttpRecorderMode` in the `TEST_CSM_ORGID_AUTHENTICATION` environment variable is consistent with the value in `AZURE_TEST_MODE` environment variable.
 
-## 7. Environment Variable Reference
+##7. Connection string
+Connection string is provided to Test Framework using following environment variables.
+In order to debug test set the following environment variables before starting Visual Studio:
 
-#### 7.1.  Supported Keys in connection string
+	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};UserId={orgId};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
+
+#### 7.1 Supported Key=Value pairs in Connectionstring
```

Comment:
@shahabhijeet do we need this header if the title of the section is the same thing?